sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -345,21 +346,14 @@ class MapboxMap internal constructor(
    *
    * @return Zoom value.
    */
-  override fun getMinZoom() = nativeMapWeakRef.call { this.minZoom }
+  override fun getMinZoom() = nativeMapWeakRef.call { this.bounds.minZoom }
 
   /**
    * Get maximum zoom.
    *
    * @return Zoom value.
    */
-  override fun getMaxZoom() = nativeMapWeakRef.call { this.maxZoom }
-
-  /**
-   * The current zoom factor applied on the map meaning 2 ^ Current zoom
-   *
-   * @return Scale value.
-   */
-  override fun getScale() = nativeMapWeakRef.call { this.scale }
+  override fun getMaxZoom() = nativeMapWeakRef.call { this.bounds.maxZoom }
```

Comment:
Upstream we made the decision to not expose getMinZoom and getMaxZoom directly on map object, only through bounds API. Shouldn't we do the same here?

Reply:
I guess you're correct, will do it! I basically left them because we use those methods for camera animations but API alignment is much more important of course.